src/main/java/org/gridsuite/actions/server/ContingencyListService.java

```diff
@@ -189,20 +189,44 @@ private List<ContingencyInfos> evaluateContingencyList(PersistentContingencyList
         List<Contingency> contingencies = getPowsyblContingencies(persistentContingencyList, network);
         Map<String, Set<String>> notFoundElements = persistentContingencyList.getNotFoundElements(network);
 
+        // For a gridsuite contingency with all equipments not found the powsybl contingency is not created
         List<ContingencyInfos> contingencyInfos = new ArrayList<>();
-        // we add all the contingencies that have only wrong ids
         notFoundElements.entrySet().stream()
                 .filter(stringSetEntry -> contingencies.stream().noneMatch(c -> c.getId().equals(stringSetEntry.getKey())))
-                .map(stringSetEntry -> new ContingencyInfos(stringSetEntry.getKey(), null, stringSetEntry.getValue()))
+                .map(stringSetEntry -> new ContingencyInfos(stringSetEntry.getKey(), null, stringSetEntry.getValue(), null))
                 .forEach(contingencyInfos::add);
 
         contingencies.stream()
-                .map(contingency -> new ContingencyInfos(contingency, notFoundElements.get(contingency.getId())))
+                .map(contingency -> createContingencyInfoWithDisconnects(contingency, network, notFoundElements.getOrDefault(contingency.getId(), null)))
```

Comment:
Why getOrDefault ?

Reply:
agree , get return null by default if not found